client/src/Movies/MovieList.js

```diff
@@ -1,5 +1,7 @@
 import React from 'react';
 
+import {Link} from 'react-router-dom';
```

Comment:
You should run Prettier on your code for better and more consistent formatting

Reply:
okay will start being more consistent . thank you for your advice.

Reply:
https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

Reply:
I assume you use Vscode?

In that case, you can find it here:

https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode